package.json

```
@@ -90,10 +91,14 @@
     "@emotion/react": "^11.4.1",
     "@emotion/styled": "^11.3.0",
     "framer-motion": ">=6.2.8",
+    "mapbox-gl": "2.13.0",
```

To allow bumping minor and patch versions:

```suggestion
    "mapbox-gl": "^2.13.0",
```